README.md

```diff
@@ -514,7 +514,7 @@ Thus when you create a TODO, it is almost always your name that is given.
      end
      ```
 
-* <a name="no-default-args"></a>Do not use default arguments. Use an options
+* <a name="no-default-args"></a>Do not use default positional arguments. Use named arguments or an options
```

Comment:
@aagrawal2001 , i think a major concern with default positional args is the inability to properly have multiple, as to specify the latter ones, you must provide values for the earlier ones. keyword arguments are not subject to this behavior.

positional arguments (optional/defaulted or not) generally being obscure is another question